src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -479,75 +527,33 @@ public void traverse() {
 
         final int sampleCount = sampleNameToVcfPath.size();
         final int updatedBatchSize = (batchSize == DEFAULT_ZERO_BATCH_SIZE) ? sampleCount : batchSize;
-        final int totalBatchCount = (sampleCount/updatedBatchSize) + (sampleCount%updatedBatchSize==0 ? 0 : 1);
-
+//        final int totalBatchCount = (sampleCount/updatedBatchSize) + (sampleCount%updatedBatchSize==0 ? 0 : 1);
+        final ImportConfig importConfig = createImportConfig(updatedBatchSize);
+        //TODO: Fix this.
+//        BiConsumer<Map<String, FeatureReader<VariantContext>>, Integer> closeReaders = (readers, batchCount) -> {
+//            progressMeter.update(intervals.get(0));
+//            logger.info("Done importing batch " + batchCount + "/" + totalBatchCount);
+//        };
         GenomicsDBImporter importer;
-
-        for (int i = 0, batchCount = 1; i < sampleCount; i += updatedBatchSize, ++batchCount) {
-
-            final SortedMap<String, FeatureReader<VariantContext>> sampleToReaderMap =
-                    inputPreloadExecutorService != null
-                            ? getFeatureReadersInParallel(sampleNameToVcfPath, updatedBatchSize, i)
-                            : getFeatureReadersSerially(sampleNameToVcfPath, updatedBatchSize, i);
-
-            logger.info("Importing batch " + batchCount + " with " + sampleToReaderMap.size() + " samples");
-            final long variantContextBufferSize = vcfBufferSizePerSample * sampleToReaderMap.size();
-            final GenomicsDBImportConfiguration.ImportConfiguration importConfiguration =
-                    createImportConfiguration(workspace, GenomicsDBConstants.DEFAULT_ARRAY_NAME,
-                                              variantContextBufferSize, segmentSize,
-                                              i, (i+updatedBatchSize-1),
-                                              (batchCount == 1)); //Fail if array exists and this is the first batch
-
-            try {
-                importer = new GenomicsDBImporter(sampleToReaderMap, mergedHeaderLines, intervals.get(0), validateSampleToReaderMap, importConfiguration);
-            } catch (final IOException e) {
-                throw new UserException("Error initializing GenomicsDBImporter in batch " + batchCount, e);
-            } catch (final IllegalArgumentException iae) {
-                throw new GATKException("Null feature reader found in sampleNameMap file: " + sampleNameMapFile, iae);
-            }
-            try {
-                importer.importBatch();
-            } catch (final IOException e) {
-                throw new UserException("GenomicsDB import failed in batch " + batchCount, e);
-            }
-            closeReaders(sampleToReaderMap);
-            progressMeter.update(intervals.get(0));
-            logger.info("Done importing batch " + batchCount + "/" + totalBatchCount);
+        try {
+            importer = new GenomicsDBImporter(importConfig);
+            importer.executeImport();
```

Comment:
Is this batching done internally in `executeImport()` now? If so, are there regular status messages emitted to the logger before/after each batch?

Reply:
Yes

Reply:
Correction - progress isn't printed now. Will try to figure this out

Reply:
Ok, thanks -- regular logger updates before/after each batch are crucial for us, as our `GenomicsDBImport` jobs tend to be very long-running.

Reply:
In the latest version I'm seeing a lot of 
```
16:26:48.128 INFO  GenomicsDBImport - Starting batch input file preload
16:27:11.629 INFO  GenomicsDBImport - Finished batch preload
```
but I prefer the old "batch X out of Y" format.  We have import jobs that run for 24 hours and it's nice to know how much progress has been made.

Reply:
Done

Reply:
I'm not getting any logging from querying the GDB either, e.g. running GenotypeGVCFs.

Reply:
I see lines:
```
14:46:01.216 INFO  ProgressMeter -              1:69417              0.3                  1000           3505.9
14:46:05.059 WARN  InbreedingCoeff - Annotation will not be calculated, must provide at least 10 samples
14:50:01.511 INFO  ProgressMeter -             1:876592              4.3                  7000           1631.6
14:50:13.315 INFO  ProgressMeter -             1:892516              4.5                 12000           2674.5
```
Is that adequate?

Reply:
How many samples was that?  My problem is that I seems to be getting updates every X variants, but for 80,000 samples that takes a very long time.